app/points/calculate-weighted-points.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (count, points, weighting = 1) {
```

Comment:
It no longer makes sense that is lives in its own file as it's only used in `calculate-tier-points.js`